src/NzbDrone.Core/Download/History/DownloadHistory.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using NzbDrone.Core.Datastore;
+
+namespace NzbDrone.Core.Download.History
+{
+    public class DownloadHistory : ModelBase
+    {
+        public DownloadHistoryEventType EventType { get; set; }
+        public int SeriesId { get; set; }
```

Comment:
What are we doing to do when a series gets deleted and readded? (esp when people 'panic' re-add a series to try refresh metadata)   
Previously, history would get purged in housekeeping. I'm wondering if we'd want the same here or be a bit more 'safe'.